backend/models/forms.js

```diff
@@ -6,25 +6,21 @@ const mongoose = require("mongoose");
 //file type
 //checkbox grid?
 
-const option = new mongoose.Schema({
-  min: {
-    type: Number
-  },
-  max: {
-    type: Number
-  },
-  opt: {
+const validation = new mongoose.Schema({
```

Comment:
I removed the "options" schema where minx/max was included. Instead, this will be included in validations because that is what this really is. 

Validations now has an enum

Reply:
Sounds good